mne/export/_export.py

```diff
@@ -10,11 +10,16 @@
 
 
 @verbose
-def export_raw(fname, raw, fmt='auto', physical_range='auto', verbose=None):
+def export_raw(fname, raw, fmt='auto', physical_range='auto',
+               add_ch_type=False, verbose=None):
     """Export Raw to external formats.
 
-    Supported formats: EEGLAB (set, uses :mod:`eeglabio`)
-    %(export_warning)s
+    Supported formats:
+
+        - EEGLAB (.set, uses :mod:`eeglabio`)
+        - EDF (.edf, uses ``EDFlib-Python``)
```

Comment:
What about the `:mod:` prefix? Why is it used for `eeglabio` but not for `EDFlib-Python`?

Reply:
Because eeglabio has sphinx generated docs, but EDFlib-Python does not